packages/ListBox/src/components/Footer/Footer.styles.js

```diff
@@ -1,16 +1,18 @@
 import styled from "styled-components";
+import tokens from "@paprika/tokens";
 
 export const Footer = styled.div`
-  border-top: 1px solid #d7d7d7;
+  align-items: center;
+  border-top: 1px solid ${tokens.color.blackLighten60};
   display: flex;
+  height: 31px;
```

Comment:
instead of setting height to 31px, we can set height to 48px (or 6 * $space)
and set `box-sizing: border-box`.